data/map.txt

```diff
@@ -2108,6 +2108,8 @@ system Aescolanus
 	attributes "ember waste" "wormhole"
 	habitable 233.28
 	belt 1323
+	hazard "EW Base Heat" 100
+	hazard "EW Base Storm" 4000
```

Comment:
This means that a storm hazard will appear on average every 4000 frames. Each storm hazard lasts on average (1500+300)/2 = 900 frames. That means that the Ember Waste will have a storm active 900/4000 = 22.5% of the time. That seems a bit frequent to me. The length of the storms seems fine, so how about reducing their spawn frequency to at least 9000. That means that 900/9000 = 10% of the time there'd be a storm.

Reply:
OK. Seems reasonable. Dropped the frequency down to 9000, being about 10%. Combined with the drop in strength should make them a lot less of a problem, while still being potentially a heavy impact.